MIGRATING.md

```diff
@@ -319,6 +319,17 @@ Below methods have been added.
 - `ForgotPasswordResponse forgotPasswordSetNewPassword(UserCredentials userCredentials, Boolean sendEmail)`
 - `ForgotPasswordResponse forgotPasswordSetNewPassword(UserCredentials userCredentials)`
 
+### Package `com.okta.sdk.resource.user.UserBuilder`
+
+Below method has undergone a signature change.
+- `UserBuilder setProvider(Boolean provider)` signature changed to `UserBuilder setProvider(AuthenticationProvider provider)`
+
+### Package `com.okta.sdk.resource.impl.DefaultUserBuilder`
```

Comment:
We can skip this section because we already noted the interface level changes in the section above.